docs/conf.py

```diff
@@ -142,246 +82,155 @@
+    'sphinx.ext.autodoc',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.todo',
+    'sphinx.ext.coverage',
+    'sphinx.ext.inheritance_diagram',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.doctest',
+    'sphinx.ext.mathjax',
+    'sphinx_automodapi.automodapi',
+    'sphinx_automodapi.smart_resolver',
+    'sphinx_gallery.gen_gallery',
+    'sunpy.util.sphinx.doctest',
+    'matplotlib.sphinxext.plot_directive',
+    'sunpy.util.sphinx.minigallery',
+    'sunpy.util.sphinx.generate',
+    'sunpy.util.sphinx.changelog',
+]
 
-def fix_circleci(x): return None
-
-
-try:
-    from sunpy_sphinx_theme.conf import *
-except ImportError:
-    html_theme = 'default'
-
-try:
-    import ruamel.yaml as yaml
-    has_yaml = True
-    # Load data about stability
-    with open('./dev_guide/sunpy_stability.yaml', 'r') as estability:
-        sunpy_modules = yaml.load(estability.read(), Loader=yaml.Loader)
-
-    html_context = {
-        'sunpy_modules': sunpy_modules
-    }
-
-    def rstjinja(app, docname, source):
-        """
-        Render our pages as a jinja template for fancy templating goodness.
-        """
-        # Make sure we're outputting HTML
-        if app.builder.format != 'html':
-            return
-        src = source[0]
-        if "Current status" in src[:20]:
-            rendered = app.builder.templates.render_string(
-                src, app.config.html_context
-            )
-            source[0] = rendered
-except ImportError:
-    has_yaml = False
-    html_context = {}
-    print('Warning: Stability of SunPy API page of the documentation requires the ruamel.yaml package to be installed')
-
-# The name of an image file (within the static path) to use as favicon of the
-# docs. This file should be a Windows icon file (.ico) being 16x16 or 32x32
-# pixels large.
-html_favicon = "./logo/favicon.ico"
-
-# The name for this set of Sphinx documents.  If None, it defaults to
-# "<project> v<release> documentation".
-html_title = f'{project} v{release}'
-
-# Output file base name for HTML help builder.
-htmlhelp_basename = project + 'doc'
-
-# A dictionary of values to pass into the template engine’s context for all pages.
-html_context['to_be_indexed'] = ['stable', 'latest']
-
-# -- Options for LaTeX output --------------------------------------------------
-# Grouping the document tree into LaTeX files. List of tuples
-# (source start file, target name, title, author, documentclass [howto/manual]).
-latex_documents = [('index', project + '.tex', project + ' Documentation', author, 'manual')]
-
-# -- Options for manual page output --------------------------------------------
-# One entry per manual page. List of tuples
-# (source start file, name, description, authors, manual section).
-man_pages = [('index', project.lower(), project + ' Documentation', [author], 1)]
-
-# -- Swap to Napoleon ---------------------------------------------------------
-# Remove numpydoc
-extensions.remove('numpydoc')
```

Comment:
Where did these go? Is napoleon config not needed any more>?

Reply:
Good question, I assumed we never needed to do this in the first place it was just some legacy config we left over. I have no idea what it's even done to the docs now its not been removed.

Reply:
Yeah, I think the below napoleon lines need copying across.

Reply:
Do you know what the difference is? I can't seem to find any in our docs at a glance.

Reply:
Just had a google.